  <launch>
    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

    <node pkg="practica_acciona" type="start_snake_service.py" name="start_snake_service"/>

    <node name="teleop" pkg="turtlesim" type="turtle_teleop_key"  output="screen" launch-prefix="xterm -e"/>

    <node name="turtle1_broadcaster" pkg="practica_acciona" type="turtle_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>

  </launch>